aegea/deploy.py

```diff
@@ -81,8 +81,15 @@ def configure(args):
                                        aws_key=key.id,
                                        aws_secret=key.secret))
     logger.info("Created SNS topic %s and GitHub hook for repo %s", topic, repo)
-    status_bucket = resources.s3.create_bucket(Bucket="deploy-status-" + ARN(topic.arn).account_id)
-    logger.info("Created %s", status_bucket)
+
+    status_bucket_name = "deploy-status-" + ARN(topic.arn).account_id
+    try:
+        resources.s3.meta.client.head_bucket(Bucket=status_bucket_name)
+    except ClientError:  #if the bucket does not already exist
```

Comment:
This exception isn't specific to the bucket not existing, see http://boto3.readthedocs.io/en/latest/guide/migrations3.html

```
# Boto 3
import botocore
bucket = s3.Bucket('mybucket')
exists = True
try:
    s3.meta.client.head_bucket(Bucket='mybucket')
except botocore.exceptions.ClientError as e:
    # If a client error is thrown, then check that it was a 404 error.
    # If it was a 404 error, then the bucket does not exist.
    error_code = int(e.response['Error']['Code'])
    if error_code == 404:
        exists = False
```

Reply:
@midnighteuler according to this, what should we do if it is not 404?